### PR TITLE
feat/added-reviews-table

### PR DESCRIPTION
**Pull request by NathanAmanuelG**

### CODE REVIEW

File: ReviveIT/Domain/Configurations/ReviewsConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+
```
Comment (donatgosalcii) [inline ReviveIT/Domain/Configurations/ReviewsConfiguration.cs:5]: Remove empty line

File: ReviveIT/Domain/Configurations/ReviewsConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+namespace Domain.Configurations
+{
+    public class ReviewsConfiguration : IEntityTypeConfiguration<Reviews>
+    {
+        public void Configure(EntityTypeBuilder<Reviews> builder)
+        {
+            builder.HasKey(r => r.ReviewID);
```
Comment (donatgosalcii) [inline ReviveIT/Domain/Configurations/ReviewsConfiguration.cs:12]: Add a new line for each parameter

File: ReviveIT/Domain/Entities/Reviews.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+
```
Comment (donatgosalcii) [inline ReviveIT/Domain/Entities/Reviews.cs:4]: Remove empty line

File: ReviveIT/Infrastructure/Data/ApplicationDbContext.cs [hunk 4]
```diff
@@ -21,7 +21,8 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
         public DbSet<Service> Services { get; set; }
         public DbSet<Subscriptions> Subscriptions { get; set; }
         public DbSet<Messages> Messages { get; set; }
-        
+
```
Comment (donatgosalcii) [inline ReviveIT/Infrastructure/Data/ApplicationDbContext.cs:24]: Remove empty line

File: ReviveIT/Infrastructure/Data/ApplicationDbContext.cs [hunk 5]
```diff
@@ -31,6 +32,7 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.ApplyConfiguration(new ServicesConfigurations());
             builder.ApplyConfiguration(new SubscriptionsConfiguration());
             builder.ApplyConfiguration(new MessagesConfigurations());
+            builder.ApplyConfiguration(new ReviewsConfiguration()); // Nathan
```
Comment (donatgosalcii) [inline ReviveIT/Infrastructure/Data/ApplicationDbContext.cs:35]: Remove the name from comments (//Nathan)

**Review by donatgosalcii — CHANGES_REQUESTED** [review]

File: ReviveIT/Infrastructure/Data/ApplicationDbContext.cs (hunk 5 above)
Comment (NathanAmanuelG) [reply ReviveIT/Infrastructure/Data/ApplicationDbContext.cs:35]: done

File: ReviveIT/Domain/Entities/Reviews.cs (hunk 3 above)
Comment (NathanAmanuelG) [reply ReviveIT/Domain/Entities/Reviews.cs:4]: ok

File: ReviveIT/Domain/Entities/Reviews.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        public DateTime CreatedAt { get; set; }
+        public DateTime UpdatedAt { get; set; }
+
+        public string UserId { get; set; }
+        public Users User { get; set; }
+
+        public int ServiceID { get; set; }
+        public Service Service { get; set; }
+    }
+    // Nathan
```
Comment (betimtasholli) [inline ReviveIT/Domain/Entities/Reviews.cs:19]: Remove Comments like this

Comment (NathanAmanuelG) [reply ReviveIT/Domain/Entities/Reviews.cs:19]: donat already addressed this point

**Review by donatgosalcii — APPROVED** [review]
Requested changes complete.

**Review by betimtasholli — APPROVED** [review]

**Pull request merged**